ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -0,0 +1,121 @@
+base_images:
+  base_fedora:
+    name: fedora
+    namespace: openshift
+    tag: "30"
```

Comment:
F30 is EOL for half a year already, is this intentional?

Reply:
No, it is not. I tried to switched to fedora:33, but I am getting this error now:

```
could not resolve inputs: could not determine inputs for step [input:base_fedora]: could not resolve base image: imagestreamtags.image.openshift.io "fedora:33" not found
```

Which image should I use? Or does fedora:33 need to be registered somewhere (did look, but could not find it)?

Reply:
I don't know who controls images in `openshift`, sorry.

Reply:
used fedora:31 which is the last one that worked for me. Anyway older version should not be a problem since this is just a throwaway CI test image and not a release one.